src/commands/shell.js

```diff
@@ -68,25 +69,32 @@ function startShell(client, endpoint, dbscope, log) {
       return cb()
     }
     defaultEval(cmd, ctx, filename, function (error, result) {
+      let res = esprima.parseScript(cmd)
+
+      
       if (error) {
         if (isRecoverableError(error)) {
           return cb(new repl.Recoverable(error))
         } else {
           return cb(error, result)
         }
       } else {
-        return client.query(result)
-        .then(function (response) {
+        return runQueries(res.body, client)
+        .then(res => {
           // we could provide the response result as a second
           // argument to cb(), but the repl util.inspect has a
           // default depth of 2, but we want to display the full
           // objects or arrays, not things like [object Object]
-          console.log(util.inspect(response, {depth: null}))
+          console.log(util.inspect(res, {depth: null}))
           return cb(error)
         })
-        .catch(function (error) {
+        .catch(error => {
           ctx.lastError = error
-          log('Error:', error.message)
+          log('Error:', error.faunaError.message)
+          console.log(util.inspect(JSON.parse(error.faunaError.requestResult.responseRaw), {
+            depth: null,
+            compact: false
+          }))
```

Comment:
Did you mean to leave this console.log in?

Reply:
It looks a missed debug thing 👀

Reply:
@danieltodonnell @BrunoQuaresma This is what logs out the details of the error (highlighted in the screenshot below). Line 93 simple logs out `Error: error message`, but not the details. This is similar to what we do on line 88 for successful queries.

![75267441-e85dea00-57fe-11ea-9003-7cfd0e83c258](https://user-images.githubusercontent.com/59929076/76426339-4eab4680-6381-11ea-9baa-18f6d202d17f.png)